Dockerfile.py3

```diff
@@ -107,6 +107,7 @@ RUN dpkg --add-architecture i386 \
         libpangox-1.0-0:i386 libpangoxft-1.0-0:i386 libidn11:i386 \
         zip zlib1g-dev zlib1g:i386 \
     && apt -y autoremove
+RUN ${RETRY} pip3 install -U Cython
```

Comment:
Wait what? It's already there https://github.com/kivy/python-for-android/blob/v2019.07.08/Dockerfile.py3#L126